src/index.html

```diff
@@ -3,12 +3,76 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin=" "
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="header-logo">
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          class="logo-image"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a href="#" class="nav-link is-active">
+              Apple
+            </a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link">
+              Samsung
+            </a>
+          </li>
```

Comment:
add blanks between multiline siblings
```suggestion
          <li class="nav-item">
            <a href="#" class="nav-link is-active">
              Apple
            </a>
          </li>
          
          <li class="nav-item">
            <a href="#" class="nav-link">
              Samsung
            </a>
          </li>
```